contrib/cf-remote/cf_remote/main.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+
+from cf_remote import log
+from cf_remote import commands
+from cf_remote.utils import user_error
+
+
+def get_args():
+    ap = argparse.ArgumentParser(
+        description="Spooky CFEngine at a distance",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
Spooky? ;)

Reply:
@craigcomstock A reference to https://www.google.com/search?q=spooky+action+at+a+distance

Reply:
Oh dear. You remind me that I never took physics. :p Very well. :)